lib/driver-helper.js

```diff
@@ -530,6 +530,26 @@ export async function dismissAlertIfPresent( driver ) {
 	}
 }
 
+export async function clearCache( driver ) {
+	let url = await driver.getCurrentUrl();
+	if ( url.startsWith( 'data:' ) === true || url === 'about:blank' ) {
+		return;
+	}
+	// Clear the cookies and cache for the ChromeDriver instance
+	// navigate to the settings page
+	await driver.get( 'chrome://settings/clearBrowserData' );
+	const clearCacheSelector = by.css( '* /deep/ #clearBrowsingDataConfirm' );
+
+	// wait for the button to appear
+	await isEventuallyPresentAndDisplayed( driver, clearCacheSelector );
+
+	// click the button to clear the cache
+	await clickWhenClickable( driver, clearCacheSelector );
+
+	// wait for the button to be gone before returning
+	await driver.sleep( 10000 );
```

Comment:
It really takes a while to go away sometimes, not sure of a better way than to just wait here

Reply:
driverHelper.waitTillNotPresent()?

Reply:
Thanks! It worked

Reply:
@bsessions85 I think this could've actually been a great use casse for Github's new code suggestion feature, drawback is that it bypasses the linter